src/main/java/chalmers/pimp/model/canvas/layer/Doodle.java

```diff
@@ -241,4 +235,18 @@ private int getExtreme(List<? extends Integer> list,
     }
     return extreme;
   }
+
+  @Override
+  public void addPoint(Point p) {
+    points.add(p);
+  }
+
+  @Override
+  public void removePoint(Point p, double threshold) {
```

Comment:
null point

Reply:
requireNotNull is called in the .distance method